kotlinx-coroutines-core/native/src/Dispatchers.kt

```diff
@@ -4,18 +4,18 @@
 
 package kotlinx.coroutines
 
+import kotlinx.coroutines.internal.multithreadingSupported
 import kotlin.coroutines.*
 
-/** Not inside [Dispatchers], as otherwise mutating this throws an `InvalidMutabilityException`. */
```

Comment:
Is this no longer true for some reason, or do we not want to support `Dispatchers.setMain` on Native without the new memory model?